.trampolinerc

```diff
@@ -21,6 +21,8 @@ required_envvars+=(
 # Add env vars which are passed down into the container here.
 pass_down_envvars+=(
     "AUTORELEASE_PR"
+    "ENVIRONMENT"
+    "RUNTIME"
```

Comment:
I think this file might be templated by synthtool. I had to [do this](https://github.com/googleapis/python-logging/blob/9069738923a498a88ef69eb6931a976cbd550418/synth.py#L77) to prevent it from overwriting

Reply:
pro tip, thanks